src/_pytest/runner.py

```diff
@@ -153,9 +153,9 @@ def pytest_runtest_teardown(item, nextitem):
 
 def _update_current_test_var(item, when):
     """
-    Update PYTEST_CURRENT_TEST to reflect the current item and stage.
+    Update :envvar:`PYTEST_CURRENT_TEST` to reflect the current item and stage.
 
-    If ``when`` is None, delete PYTEST_CURRENT_TEST from the environment.
+    If ``when`` is None, delete ``PYTEST_CURRENT_TEST`` from the environment.
```

Comment:
`:envvar:` here too :wink:

Reply:
@Zac-HD 
Thanks for the review!
IIRC it is a matter of only linking the first occurrence in a paragraph/section, but not all of them.
Please (squash-)merge if that sounds OK to you.

Reply:
For Hypothesis we prefer to use the role syntax for every occurrence so that they're all styled and linked, but happy for you to merge now if you disagree.

Reply:
I do not really disagree, but rather take it as-is for now, thanks!